go/service/chat_local.go

```diff
@@ -205,87 +202,36 @@ func (h *chatLocalHandler) GetInboxSummaryLocal(ctx context.Context, arg keybase
 			return res, fmt.Errorf("parsing time or duration (%s) error: %s", arg.Since, since)
 		}
 	}
-	since2 := time.Now().Add(-time.Since(since) * 2)
 
-	// TODO: move these criterian to server using queries
-	appendMaybe := func(conv chat1.Conversation) (shouldContinue bool, err error) {
+	var rpcArg keybase1.GetInboxLocalArg
+	if arg.Limit != 0 {
+		rpcArg.Pagination = &chat1.Pagination{Num: arg.Limit}
+	}
+	var query chat1.GetInboxQuery
+	if !since.IsZero() {
+		gsince := gregor1.ToTime(since)
+		query.Before = &gsince
+	}
+	if len(arg.TopicTypes) > 0 {
+		query.TopicType = &arg.TopicTypes[0]
```

Comment:
Yeah the CLI design page made it look like only one at a time was the way, so that is how I did it in the server API. I got here and saw you had a list, so just took the easy way out...